python/paddle/fluid/device_worker.py

```diff
@@ -404,17 +404,16 @@ def _gen_worker_desc(self, trainer_desc):
         pipeline_opt = self._program._pipeline_opt
         section_param = trainer_desc.section_param
         section_param.queue_size = pipeline_opt["queue_size"]
-        section_param.sync_steps = pipeline_opt["sync_steps"]
+        section_param.microbatch = pipeline_opt["queue_size"]
```

Comment:
maybe just microbatch is enough, we don't need a new concept for configuration, namely queue_size.

Reply:
even though this is generated through inner API:)

Reply:
Done.